### ISSUE TITLE
Download an article should also download the PDF

### ISSUE THREAD

**Issue by koppor** [JabRef/jabref#12889]
1. I fetch `arXiv:2201.00682v1` 
2. An entry with a http link in the file field generated

Wish: the file should be downloaded automatically.

This refs https://github.com/koppor/jabref/issues/568, because "manual" downloading should be easy

**Comment by Siedlerchr** [comment]
I think this issue is obsolete. By default and if the preferences is activated 
https://arxiv.org/abs/2201.00682  it will download

<img width="747" alt="Image" src="https://github.com/user-attachments/assets/49c2e673-67ef-4da9-a377-12c3acfa6734" />

**Comment by koppor** [comment]
@madanhk18 could you test maybe?

**Comment by madanhk18** [comment]
<img width="1417" alt="Image" src="https://github.com/user-attachments/assets/c7447bfb-2518-4a9f-8c7e-b4bd6d1a7e7a" />

Sir in the entry editor , It is downloading the PDF . Is this all or need to edit something sir? Actually i was working on this issue but when i saw now it says it is closed

**Comment by koppor** [comment]
@madanhk18 I see a http link only, no PDF. Do you have activated "Download referenced files"?

**Comment by madanhk18** [comment]
<img width="734" alt="Image" src="https://github.com/user-attachments/assets/90913e2a-0a76-4b00-a112-d7ea87a8e979" />

Yes sir I have checked the box "Download referenced files

**Comment by InAnYan** [comment]
I also don't have a PDF. @madanhk18, you can continue working on this issue 😄

**Comment by Siedlerchr** [comment]
How did you add the entry? Where exactly did you paste it?

Ruslan ***@***.***> schrieb am Mo., 7. Apr. 2025, 07:06:

**Comment by InAnYan** [comment]
Example: <https://arxiv.org/abs/2501.00117>

**Comment by madanhk18** [comment]
> Example: https://arxiv.org/abs/2501.00117

Sir Can you please calrify , that PDF should be automatically download or in that entry editor in General section it is anyhow showing the pdf http link

**Comment by InAnYan** [comment]
Sorry, right question!

I expected that JabRef would both download the PDF and include the PDF as a linked file.

Currently, JabRef doesn't download the PDF and the "linked file" is actually just an HTTP link to the PDF

**Comment by madanhk18** [comment]
yes sir for that It is showing the HTTP link

**Comment by Siedlerchr** [comment]
@InAnYan  Exactly what I did yesterday and for me, it downloaded the file. Did you have the Preferences set? Working directory configured?

**Comment by InAnYan** [comment]
@Siedlerchr, ah, "untitled" library just has a link to PDF -- is this right behavior or not?

"Saved" library has a PDF downloaded -- expected.

If the answer to the first question is yes, then we can close the issue

**Comment by Siedlerchr** [comment]
Yes I think this is correct behaviour. If the library is not saved we cannot determine a saving directory correctly

**Comment by koppor** [comment]
In case there is a global directory configured in the preferences, this should be used. - The other setting than "store next to bib file"

Meaning: Even for unsaved we sometimes have a path!
